huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_app_group_authorization_notification_resend_test.go

```diff
@@ -0,0 +1,126 @@
+package workspace
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccAppGroupAuthorizationNotificationResend_basic(t *testing.T) {
+	var (
+		rName                  = "huaweicloud_workspace_app_group_authorization_notification_resend.test"
+		withNotificationRecord = "huaweicloud_workspace_app_group_authorization_notification_resend.with_notification_record"
+
+		name = acceptance.RandomAccResourceName()
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		// This resource is a one-time action resource and there is no logic in the delete method.
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppGroupAuthorizationNotificationResend_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(rName, "records.#", "1"),
+					resource.TestMatchResourceAttr(withNotificationRecord, "records.#", regexp.MustCompile(`^[1-9]([0-9]*)?$`)),
+				),
+			},
+		},
+	})
+}
+func testAccAppGroupAuthorizationNotificationResend_basic(name string) string {
```

Comment:
```suggestion
}

func testAccAppGroupAuthorizationNotificationResend_basic(name string) string {
```
supports a blank line in here